### ISSUE TITLE
Suggestion: Use prepublishOnly instead of prepare for running build script?

### ISSUE THREAD

**Issue by Shurelia** [ospatil/generator-node-typescript#51]
As per the [npm docs](https://docs.npmjs.com/misc/scripts), `prepare` runs both after install and before `npm publish`. I personally can't think of any situations I'd want to build the module after each install, but if that's intentional then that's cool, it's not much effort when generating a new project to change. However, if this is not the intended behavior then it would be nice to have the default be to use the `prepublishOnly` npm script instead to only run builds before `npm publish`, and not after installs.

**Comment by ospatil** [comment]
You are right. Thanks for bringing my attention to it. It's a leftover from earlier NPM versions. I'll make the change and publish a new version sometime in the next week.
